exporter/awsemfexporter/datapoint_test.go

```diff
@@ -486,6 +593,13 @@ func TestGetDataPoints(t *testing.T) {
 			expectedDatapointSlice: histogramDataPointSlice{cumulativeDeltaMetricMetadata, pmetric.HistogramDataPointSlice{}},
 			expectedAttributes:     map[string]interface{}{"label1": "value1"},
 		},
+		{
+			name:                   "ExponentialHistogram",
+			isPrometheusMetrics:    false,
+			metric:                 generateTestExponentialHistogramMetric("foo"),
+			expectedDatapointSlice: exponentialHistogramDataPointSlice{cumulativeDeltaMetricMetadata, pmetric.ExponentialHistogramDataPointSlice{}},
```

Comment:
This attribute appears to only be used for a type assertion.  Tests still need to be added to the type switch below.

Reply:
Good catch. The test is added.